integrator/integrator.py

```diff
@@ -0,0 +1,98 @@
+from sys import argv
+from subprocess import Popen
+import os
+from integratorlib import *
+
+
+# Pack size for integrator
+kmax=1000
+
+# Checking input format
+lenarg=len(argv)
+if lenarg < 2 : quit('Error! No arguments found. Please follow the format described in documentation')
+# If only one asteroid in input
+if lenarg == 2 :
+    astlist=[argv[1]]
+# If "-list" or "-range" or neither keys are specified
+if lenarg > 2 :
+    if argv[1]=='-list':
+        astlist=argv[2:]
+    elif argv[1]=='-range':
+        try:
+            astlist=[str(i) for i in range(int(argv[2]),int(argv[3])+1)]
+        except:
+            quit('Error! Wrong range arguments. Please follow the format described in documentation')
```

Comment:
quit is bad option for python software. Throw & process an exception instead of that & return the proper results in a function.